diff --git a/src/modules/Modules.cpp b/src/modules/Modules.cpp
index e2a4a970c..89c46e479 100644
--- a/src/modules/Modules.cpp
+++ b/src/modules/Modules.cpp
@@ -164,8 +164,6 @@ void setupModules()
         }
 #endif
 
-        cardKbI2cImpl = new CardKbI2cImpl();
-        cardKbI2cImpl->init();
 #ifdef INPUTBROKER_MATRIX_TYPE
         kbMatrixImpl = new KbMatrixImpl();
         kbMatrixImpl->init();
