diff --git a/variants/esp32s3/heltec_v3/platformio.ini b/variants/esp32s3/heltec_v3/platformio.ini
index af0854e49..5c4cdec81 100644
--- a/variants/esp32s3/heltec_v3/platformio.ini
+++ b/variants/esp32s3/heltec_v3/platformio.ini
@@ -8,3 +8,14 @@ build_flags =
   ${esp32s3_base.build_flags}
   -D HELTEC_V3
   -I variants/esp32s3/heltec_v3
+  -DMESHTASTIC_EXCLUDE_MQTT=1
+  -DMESHTASTIC_EXCLUDE_ATAK=1
+  -DMESHTASTIC_EXCLUDE_ADMIN=1
+  -DMESHTASTIC_EXCLUDE_AUDIO=1
+  -DMESHTASTIC_EXCLUDE_REMOTEHARDWARE=1
+  -DMESHTASTIC_EXCLUDE_EXTERNALNOTIFICATION=1
+  -DMESHTASTIC_EXCLUDE_DETECTIONSENSOR=1
+  -DMESHTASTIC_EXCLUDE_CANNEDMESSAGES=1
+  -DMESHTASTIC_EXCLUDE_RANGETEST=1
+  -DMESHTASTIC_EXCLUDE_PAXCOUNTER=1
+  -DMESHTASTIC_EXCLUDE_DROPZONE=1
\ No newline at end of file
diff --git a/variants/esp32s3/heltec_v4/platformio.ini b/variants/esp32s3/heltec_v4/platformio.ini
index 4ff7ff253..7ae55697f 100644
--- a/variants/esp32s3/heltec_v4/platformio.ini
+++ b/variants/esp32s3/heltec_v4/platformio.ini
@@ -7,6 +7,17 @@ build_flags =
   ${esp32s3_base.build_flags}
   -D HELTEC_V4
   -I variants/esp32s3/heltec_v4
+  +  -DMESHTASTIC_EXCLUDE_MQTT=1
++  -DMESHTASTIC_EXCLUDE_ATAK=1
++  -DMESHTASTIC_EXCLUDE_ADMIN=1
++  -DMESHTASTIC_EXCLUDE_AUDIO=1
++  -DMESHTASTIC_EXCLUDE_REMOTEHARDWARE=1
++  -DMESHTASTIC_EXCLUDE_EXTERNALNOTIFICATION=1
++  -DMESHTASTIC_EXCLUDE_DETECTIONSENSOR=1
++  -DMESHTASTIC_EXCLUDE_CANNEDMESSAGES=1
++  -DMESHTASTIC_EXCLUDE_RANGETEST=1
++  -DMESHTASTIC_EXCLUDE_PAXCOUNTER=1
++  -DMESHTASTIC_EXCLUDE_DROPZONE=1
 lib_deps = 
   ${esp32s3_base.lib_deps}
 
