diff --git a/variants/esp32s3/heltec_v3/platformio.ini b/variants/esp32s3/heltec_v3/platformio.ini
index b521e11ca..ccd54221d 100644
--- a/variants/esp32s3/heltec_v3/platformio.ini
+++ b/variants/esp32s3/heltec_v3/platformio.ini
@@ -9,3 +9,5 @@ build_flags =
   -D HELTEC_V3
   -I variants/esp32s3/heltec_v3
   -D GPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -DMESHTASTIC_EXCLUDE_WIFI=1
+  -DMESHTASTIC_EXCLUDE_MQTT=1
+  -DMESHTASTIC_EXCLUDE_ATAK=1
+  -DMESHTASTIC_EXCLUDE_ADMIN=1
+  -DMESHTASTIC_EXCLUDE_I2C=1
\ No newline at end of file
